helpers/format-website.js

```diff
@@ -0,0 +1,5 @@
+function formatWebsite(website) {
+  return website.startsWith('http') ? website : 'http://' + website;
```

Comment:
So, the only website link that's working as intended is Pleasant House Pub. The others aren't working for me. 

- Restaurants on the homepage are linking out to 127.0.0.1:8888/website.com
- Restaurants on the hit list page are linking out to 127.0.0.1:8888/hit-list/website.com
- In both places, Pleasant House Pub is correctly linking out to www.pleasanthousepub.com

Maybe this is the culprit? It seems like you're only appending `http://` to websites if they're coming out of  Trello with the http already in there. So, if that's the case, the error makes sense. I entered Pleasant House's website in Trello as `http://www.pleasanthousepub.com`, but in looking at the other restaurant cards, they're all entered in Trello as `www.website.com` without the http. (Follow up: I updated Longman & Eagle's website to include `http://`, and it works as intended.)